docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -312,8 +310,8 @@ workflows:
 ## Other providers
 
 To run Semgrep CI on any other provider,
-use the `returntocorp/semgrep-agent:v1` Docker image,
-and run the `semgrep-agent` command.
+use the `returntocorp/semgrep` image,
```

Comment:
@adamkvitek not sure how best to say it but really for any provider the only thing that is important is that they run `semgrep ci` how they do so can either be with docker or they can even `pip install semgrep` if they want

Reply:
Duplicate comment, resolving (we can sort this out in a comment above).